crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1762,6 +1797,84 @@ impl<T: R2D2Connection> IndexerStore for PgIndexerStore<T> {
     fn as_any(&self) -> &dyn StdAny {
         self
     }
+
+    /// Persist protocol configs and feature flags until the protocol version for the latest epoch
+    /// we have stored in the db, inclusive.
+    async fn persist_protocol_configs_and_feature_flags(&self) -> Result<(), IndexerError> {
+        // Fetch the chain id from db. We need a while loop here because it's possible that
+        // the first checkpoint has not beed indexed yet, so we need to wait for it.
+        let mut chain_id_opt;
+        while {
+            chain_id_opt = self.get_chain_identifier()?;
+            chain_id_opt.is_none()
+        } {}
+        let chain_id = ChainIdentifier::from(
+            CheckpointDigest::try_from(chain_id_opt.unwrap()).expect("Unable to convert chain id"),
+        );
+
+        // We only store protocol configs until the version for the latest epoch, because config
+        // values for future epochs, even when they already exist, are subject to change.
+        let end_version = self.get_latest_epoch_protocol_version()?.unwrap_or(1);
+        let mut all_configs = vec![];
+        let mut all_flags = vec![];
+
+        // Protocol version starts from 1.
+        let start_version = self.get_latest_protocol_config_version()?.unwrap_or(0) + 1;
```

Comment:
I believe we are storing them in the db the same way. Perhaps this line was not written clearly enough. Do you think something like this would be better?
```rust
let start_version =
    if let Some(latest_version_in_db) = self.get_latest_protocol_config_version()? {
        // We start indexing from the next version
        latest_version_in_db + 1
    } else {
        // Nothing has been indexed yet so we start from the initial version, which is 1
        1
    };
```

Reply:
Ahh gotcha, then how about
```
let start_version = self.get_latest_protocol_config_version()?.map_or(1, |v| v + 1);
```